src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -23,7 +25,10 @@ const mpuStyles = css`
   min-height: ${MPU_HEIGHT};
 `;
 
-const AdContainer = styled.div`
+const AdContainer = styled.section.attrs({
+  ariaHidden: 'true',
```

Comment:
not sure I see this in the snapshot?

Reply:
I now changed this so that `aria-hidden` is added directly and not through `.attrs`. Snapshots now include `ariaHidden: 'true'` 👍  Thank you for spotting this